### ISSUE TITLE
investigate why some of the aria functionality doesn't work in the REPL 

### ISSUE THREAD

**Issue by isaacHagoel** [isaacHagoel/svelte-dnd-action#134]
seems like in the REPL some of the aria functionality doesn't work.
The reader ignores the instructions and alerts.
In a real app the same code works fine.
investigate why.

**Comment by isaacHagoel** [comment]
Closing because there is not enough current reproducible information to take action and this report predates substantial changes now present in v0.9.77. If the problem still occurs, please open a new focused issue with browser, Svelte and library versions plus a minimal reproduction.
